app/src/release/res/values/manifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- This is the SHA-256 digest for the Authenticator App Release variant:-->
```

Comment:
Should we clarify that this is the release variant using the Google Play signing key?

Reply:
Yes! I like that idea.

7ca58e2097f2251a605a38488b698d9ee34833f1